src/js/components/modals/CliInstallModal.tsx

```diff
@@ -67,13 +50,7 @@ class CliInstallModal extends React.Component {
     }
     const clusterUrl = `${protocol}://${hostname}${port}`;
     const { selectedOS } = this.state;
-
-    // Starting from DC/OS >=2.0, users can stick to the "latest" CLI.
-    // However on DC/OS 1.x we must use the correct CLI for the given version of DC/OS.
-    const dcosVersion = MetadataStore.parsedVersion;
-    const cliVersion =
-      dcosVersion.substring(0, 2) === "1." ? `dcos-${dcosVersion}` : "latest";
-    const downloadUrl = `https://downloads.dcos.io/binaries/cli/${osTypes[selectedOS]}/x86-64/${cliVersion}/dcos`;
+    const downloadUrl = `https://downloads.dcos.io/binaries/cli/${osTypes[selectedOS]}/x86-64/latest/dcos`;
```

Comment:
Can we do that?
```
// However on DC/OS 1.x we must use the correct CLI for the given version of DC/OS.
```

Reply:
most likely we can't! thanks for catching!